src/main/java/marquez/db/DatasetDao.java

```diff
@@ -70,7 +69,7 @@ default void insertAll(
   @SqlUpdate(
       "UPDATE datasets "
           + "SET updated_at = NOW(), "
-          + "    current_version_uuid = :currentVersionUuid "
```

Comment:
The space here was added for alignment and readability. Why remove it?

Reply:
Ohh ok, looked a little unnatural to me, but I can move it back.